### ISSUE TITLE
Choice of training data

### ISSUE THREAD

**Issue by mdsmith-cim** [valeoai/bravo_challenge#6]
Hi,

In last year's competition, we saw that the models that generally did the best were those that were trained on a lot of data external to the datasets specified in each track, even if it is in an indirect fashion such as DINOv2 being self-supervised on LVD-142M which itself contains ADE20K, Cityscapes, ImageNet, etc. Furthermore, the vast majority of non-foundation models will usually initialize training with an ImageNet model as well.

In terms of getting the best results, this is obviously ideal. However, I feel that allowing large amounts of external data to be used even if indirectly rather undermines the competition restrictions that we must "train your models exclusively on datasets <ABC>" and makes it difficult to draw any meaningful conclusions related to the track limitations on model generalization from limited data.

For 2025, and especially for the new Task 3, can we have some additional precision on what the models are allowed to have been trained and pre-trained on?

Thanks!

**Comment by abursuc** [comment]
Hi,

Thank you for the interest in the challenge and for raising this topic on the data used for pre-training.

We have thought and discussed this within the organization team when preparing the challenge last year and this year as well.
We agree that to some extent this could encourage participants to select models trained on large datasets. 
However, there is an increasing number of foundation models that are available nowadays and with different pre-training distributions, e.g., Web-DINO, PerceptionEncoder, that were released just recently. The community is coming up with interesting ideas to leverage such models with parameter-efficient fine-tuning (e.g., Rein) or combinations of them into a single one with distillation (e.g., Theia, UNIC, RADIOv2.5, etc.) multiplying the possibilities for addressing this challenge. It would be interesting to see how such models could be used for better addressing generalization and robustness. 
On our side, so far we've put restrictions rather on the fine-tuning part for the data distributions of interest for this challenge, i.e., urban driving.

Coming back to your question, we're not sure how we should frame a more limited data regime for the pre-training. ImageNet itself is also big, curated and labeled (1.2M images), while other pre-training datasets are larger, less curated and with noisy labels. Putting a restriction on the amount of data or type of supervision during pre-training can be tricky and risks limiting the scope of ideas. 

An alternative way would be to have a track for training from scratch, but few people do this nowadays. Another idea could be to have a smaller subset of labelled data for fine-tuning (few-shot style, e.g., 10% of labelled data). Would this be of interest for you?
We welcome ideas and suggestions and would be happy to discuss this further with you.

Thank you again!
Andrei

**Comment by mdsmith-cim** [comment]
Thanks for providing such a detailed reply!

Re: various foundation models, I certainly do agree and find them a very interesting area of research. I think I was not the most clear with my wording; I'll try and explicitly list my concerns and give some suggestions for what I'd like to see addressed:

1. The challenge guidelines say we may only "train your models exclusively on datasets XYZ" ). Strictly speaking, as written this would exclude even ImageNet. As you wrote that this restriction is on fine-tuning explicitly, that statement/paragraph should be reworded to explicitly say that.
2. I think there are some interesting possibilities for a track with more stringent conditions, to see how far generalization can be pushed from limited data, but I don't have any particular suggestions.
3. My biggest concern with pre-training data is not about being more limited with data, but rather ensuring no overlap of the training and test distributions when called for. For instance, with the new Track 3, the goal seems to be testing generalization from synthetic to real data, and we're not allowed to train on Cityscapes or any real data. However, if we fine-tune on DINOv2, we get Cityscapes (among many others) in the pre-training data, thus significantly reducing the distribution shift experienced by the network. Put another way, I feel pre-training can be used as a way to semi-bypass the prohibition on allowed training datasets in the context of urban driving.

I hope this is clear! 

Michael

**Comment by abursuc** [comment]
Hi Michael,

I'm sorry for the late reply. I completely missed the notification of your response.
Thank you for your valuable suggestions and interest in this challenge.

> 1. The challenge guidelines say we may only "train your models exclusively on datasets XYZ" ). Strictly speaking, as written this would exclude even ImageNet. As you wrote that this restriction is on fine-tuning explicitly, that statement/paragraph should be reworded to explicitly say that.

Yes, that's right. We took the common practice for methods reporting performance in Cityscapes and that all use a pretrained network nowadays. The baselines we proposed are also all pretrained. 
We will rephrase that in the description to avoid potential confusions. 

> 2. I think there are some interesting possibilities for a track with more stringent conditions, to see how far generalization can be pushed from limited data, but I don't have any particular suggestions.

I guess we can try something with a limited fine-tuning dataset, as done in self-supervised or semi-supervised learning: start from a pretrained network and finetune on 1%, 5%, 10% of labeled images or something like that. I'm not sure if doing this with a randomly initialized network has much practical interest.
Would you be interested in such a task? If so, we can include something for the next edition of the challenge, for ICCV 2025.

> 3. My biggest concern with pre-training data is not about being more limited with data, but rather ensuring no overlap of the training and test distributions when called for. For instance, with the new Track 3, the goal seems to be testing generalization from synthetic to real data, and we're not allowed to train on Cityscapes or any real data. However, if we fine-tune on DINOv2, we get Cityscapes (among many others) in the pre-training data, thus significantly reducing the distribution shift experienced by the network. Put another way, I feel pre-training can be used as a way to semi-bypass the prohibition on allowed training datasets in the context of urban driving.

Indeed, this is a legitimate concern. However, while most pretrained models have likely seen Cityscapes-like images (urban streetview images), they don't have pixel-level supervision (ImageNet pretraining uses image-level class labels,  CLIP uses noisy image-level text captions as supervision, DINOv2 and other self-supervised models see large amounts of images but without labels).
The benchmark is definitely not perfect, but we're trying to push in the direction of getting things right. So, we welcome any suggestions and are grateful for your feedback so far.

Best,
Andrei

**Comment by mdsmith-cim** [comment]
Hi Andrei,

It's certainly been an interesting discussion! I appreciate the back and forth.

Re: a limited fine tuning dataset - I may be interested, although my current research focus is more on quantifying the effects of different sources (as I did in my submission last year) than determining what can be obtained from say 5% of labeled images. I am curious with foundation models exactly where the performance comes from -  is it lots of data? Lots of multi-domain data? Lots of data relevant to the task? etc. I have no real interest in determining what can be obtained from a randomly initialized network. 

The next edition of the challenge is on my radar, time permitting.

Best,

Michael

**Comment by abursuc** [comment]
Hi Michael,
Thanks again. 
Looking forward to seeing your submission for the next challenge :)

Best,
Andrei
